os-autoinst-obs-auto-submit

```diff
@@ -16,8 +16,20 @@ XMLSTARLET=$(command -v xmlstarlet || true)
 [[ -n $XMLSTARLET ]] || (echo "Need xmlstarlet" && exit 1)
 
 # shellcheck source=/dev/null
-. "$(dirname "$0")"/_common
+. "$(dirname "${BASH_SOURCE[0]}")"/_common
 
+get_submit_target_extra_latest_version() {
+    # awk iterates through all the fields in the match to avoid fixed position
+    # and it should match with the part which starts with `submit_target_extra_project`+`:`
+    latest_submit_target_extra_version=$(osc dists | grep "$submit_target_extra_project" | awk -v prj="$submit_target_extra_project" '{for (i=1; i<=NF; i++) if ($i ~ "^" prj ":") print $i}' | sort --version-sort | tail -n 1)
+    if [[ -z $latest_submit_target_extra_version ]]; then
+        echo "Project not found." && return 1
+    fi
+    # Notice that the :Update doesn't come from osc output
+    echo "$latest_submit_target_extra_version:Update"
+}
+
+submit_target_extra="{submit_target_extra:-get_submit_target_extra_latest_version}"
```

Comment:
If I execute this line in bash, I get
```
% submit_target_extra="{submit_target_extra:-get_submit_target_extra_latest_version}"
% echo $submit_target_extra 
{submit_target_extra:-get_submit_target_extra_latest_version}
```
see also the latest jenkins pipeline http://jenkins.qa.suse.de/job/submit-openQA-TW-to-oS_Fctry/4002/console